### PR TITLE
HelioWebXRPolyfill: Fix illegal return statement error.

### LINKED ISSUE #17837: Illegal return statement error on console in WebVR examples
##### Description of the problem

On WebVR examples, for example https://threejs.org/examples/#webvr_ballshooter, I see the following error on console.

```
Uncaught SyntaxError: Illegal return statement HelioWebXRPolyfill.js:7 
```

Maybe related: #17815

##### Three.js version

- [x] Dev
- [x] r110
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

### PR DESCRIPTION
**Pull request by yhuang-ml**
Fixed #17837

### CODE REVIEW

File: examples/js/vr/HelioWebXRPolyfill.js [hunk 1]
```diff
@@ -4,189 +4,189 @@
 
 if ( /(Helio)/g.test( navigator.userAgent ) && "xr" in navigator ) {
 
-	if ( 'isSessionSupported' in navigator.xr ) return;
```
Comment (Mugen87) [inline examples/js/vr/HelioWebXRPolyfill.js]: Can you please ensure to only use hard tabs in this file? It seems hard and soft tabs are mixed now which ruins proper formatting.

Comment (mrdoob) [reply examples/js/vr/HelioWebXRPolyfill.js]: Yes please. Check the `files changed` section to make sure your pr is clean:
https://github.com/mrdoob/three.js/pull/17838/files

Comment (yhuangbrowser) [reply examples/js/vr/HelioWebXRPolyfill.js]: Updated.

**Comment by mrdoob** [comment]
How about this:

```js
if ( /(Helio)/g.test( navigator.userAgent ) && 'xr' in navigator && 'isSessionSupported' in navigator.xr === false ) {
```

**Comment by yhuangbrowser** [comment]
> How about this:
> 
> ```js
> if ( /(Helio)/g.test( navigator.userAgent ) && 'xr' in navigator && 'isSessionSupported' in navigator.xr === false ) {
> ```

Done

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
@yhuangbrowser When do you think we'll be able to remove this file?

**Comment by yhuangbrowser** [comment]
> @yhuangbrowser When do you think we'll be able to remove this file?

Thanks for merging the pr. I don't have a certain timeline now but my estimation is at least 3~4 months after our next software updates.

**Pull request merged**